### DISCUSSION TITLE
Feature Request: Input tags directly in the popup instead of going into details

### DISCUSSION THREAD

**Discussion by dsimonow** [coddingtonbear/obsidian-web#116] [Ideas]
Love the Extension.
One friction in using it though is that to add tags you need to open up details everytime. 
Obsidian offers a great way to search topics by tag and making it easier to fire&forget notes would be a great addition in setting up a knowledge base about stuff you might wanna look into when you need it.

I thought maybe it could be a textfield above the details button in the popup and people can add it to their template through something like {tags} or something. Frontmatter has the tags tag but I think it would be even fine if people would just have to type them in as they would in their markdown like hashtagTitle and so on.

**Comment by coddingtonbear** [comment]
I'm afraid there are so, so, so many different and idiosyncratic ways folks might want to optimize their workflows for getting content into their vault from the web, and although this particular feature might be useful to you and some other number of users, it's probably not likely the majority of users would use such a thing, and that's the bar I'm setting for what kinds of additional UI elements we might show for all users.

That said, I'm up for discussing ways we might make the UI more easily customized on a per-user basis so users can optimize their own personal workflows for things like this if you're up for embarking on a slightly different mission than the one you've proposed.

**Comment by Eflores89** [comment]
I agree with @dsimonow , maybe you can make it an optional setting what to display for each note type, similar to how "save to notion" works: https://chromewebstore.google.com/detail/save-to-notion/ldmmifpegigmeammaeckplhnjbbpccmm

**Reply by coddingtonbear** [reply]
I'm _thinking_ about adding form fields that you can define for each template, and you'd be asked to provide data for them when you select such a template, but haven't fully thought through the mechanics of how that might work.  It does seem like it'd solve this use case, too.

**Reply by kylek29** [reply]
Although I get many users want their own workflow, would that be worth the effort?

I've used *a lot* of clipping apps for the various note taking programs (Google's various attempts, OneNote, Evernote, etc.), and they all have a very similar interface/workflow. I think mirroring that could match a majority of users, since that's what they're likely used to.


Most have something like this:
- Type (e.g. snapshot, bookmark, link .. this is basically the drop-down templates right now -- with the exception of image capture)
- Title (Auto populated from the page normally, but user editable)
- Text Input (A typing field for a quick note on what they're clipping)
- Tag Manager (some form of label/tag management, although you could expose the entire tag list from Obsidian via an API hook).

That's most of them. 

I'd be willing to consider doing the work on this if it's a serious consideration for inclusion.

**Reply by coddingtonbear** [reply]
I'm afraid I already have rough sketches around this particular feature; so I'm not sure I'm looking for help around this, but I appreciate the offer!
